tests/integration/test_script_runner.py

```diff
@@ -159,10 +158,12 @@ class BackendDependencies(enum.Enum):
     # TODO: <Alex>ALEX -- uncomment next two (2) tests once Spark in Azure Pipelines is enabled.</Alex>
     # {
     #     "user_flow_script": "tests/integration/docusaurus/connecting_to_your_data/cloud/s3/spark/inferred_and_runtime_yaml_example.py",
+    #     "data_context_dir": "tests/integration/fixtures/no_datasources/great_expectations",
     #     "extra_backend_dependencies": BackendDependencies.SPARK,
     # },
     # {
     #     "user_flow_script": "tests/integration/docusaurus/connecting_to_your_data/cloud/s3/spark/inferred_and_runtime_python_example.py",
+    #     "data_context_dir": "tests/integration/fixtures/no_datasources/great_expectations",
     #     "extra_backend_dependencies": BackendDependencies.SPARK,
     # },
```

Comment:
`data_context_dir` was added as part of attempting to get the `SPARK` tests running too. Would like to leave the directory commented-out for now, since it will be needed in the future